astropy/cosmology/core.py

```diff
@@ -512,6 +515,47 @@ def clone(self, *, meta: Optional[Mapping] = None, to_nonflat: bool = False, **k
             return self.nonflat.clone(meta=meta, **kwargs)
         return super().clone(meta=meta, **kwargs)
 
+    # ===============================================================
+
+    def __equiv__(self, other):
+        """flat-|Cosmology| equivalence. Use ``.is_equivalent()`` for actual check!
+
+        Parameters
+        ----------
+        other : `~astropy.cosmology.Cosmology` subclass instance
+            The object to which to compare for equivalence.
+
+        Returns
+        -------
+        bool or `NotImplemented`
+            `True` if 'other' is of the same class / non-flat class (e.g.
+            ``FlatLambdaCDM`` and ``LambdaCDM``) has matching parameters
+            and parameter values. `False` if 'other' is of the same class but
+            has different parameters. `NotImplemented` otherwise.
```

Comment:
```suggestion
            `True` if ``other`` is of the same class / non-flat class (e.g.
            |FlatLambdaCDM| and |LambdaCDM|) has matching parameters
            and parameter values. `False` if ``other`` is of the same class but
            has different parameters. `NotImplemented` otherwise.
```

Reply:
Done. As __equiv__ is a magic method I don't think it's ever viewable on the docs. But good IDEs might show some of these improvements.

Reply:
There's no reason not to be consistent with docstrings.